### PR TITLE
Created the tabs, as well as their corresponding fragments

### PR DESCRIPTION
**Pull request by kb-jboateng**
We should consider using fragments compared to activities,
some code may have to be moved from the activities to their corresponding fragments

### CODE REVIEW

File: app/src/main/java/cmput301/team19/lendz/NotificaitionsFragment.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link NotificaitionsFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class NotificaitionsFragment extends Fragment {
```
Comment (kylemclean) [inline app/src/main/java/cmput301/team19/lendz/NotificaitionsFragment.java]: Please rename to `NotificationsFragment`

File: app/src/main/res/layout/fragment_notificaitions.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (kylemclean) [inline app/src/main/res/layout/fragment_notificaitions.xml]: Please rename to `fragment_notifications.xml`

File: app/src/main/java/cmput301/team19/lendz/MainActivity.java [hunk 3]
```diff
@@ -1,14 +1,48 @@
...
+                    Fragment fragment = null;
+                    int itemID = item.getItemId();
+                    if (itemID == R.id.borrow) {
+                        fragment = new BorrowBookFragment();
+                    }else if (itemID == R.id.my_books) {
+                        fragment = new MyBooksFragment();
+                    }else if (itemID == R.id.notifications) {
+                        fragment = new NotificaitionsFragment();
+                    } else if(itemID == R.id.profile) {
+                        fragment = new NotificaitionsFragment();
```
Comment (kylemclean) [inline app/src/main/java/cmput301/team19/lendz/MainActivity.java]: This should probably use the `ViewUserProfileFragment`

File: app/src/main/java/cmput301/team19/lendz/AddBookFragment.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link AddBookFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class AddBookFragment extends Fragment {
```
Comment (kylemclean) [inline app/src/main/java/cmput301/team19/lendz/AddBookFragment.java:16]: We can just use the upcoming `EditBookFragment` instead.

**Review by kylemclean — CHANGES_REQUESTED** [review]

File: app/src/main/java/cmput301/team19/lendz/MainActivity.java (hunk 3 above)
Comment (kb-jboateng) [reply app/src/main/java/cmput301/team19/lendz/MainActivity.java]: That has an issues that needs to be result

Comment (kb-jboateng) [reply app/src/main/java/cmput301/team19/lendz/MainActivity.java]: resolved**

File: app/src/main/java/cmput301/team19/lendz/AddBookFragment.java (hunk 4 above)
Comment (kb-jboateng) [reply app/src/main/java/cmput301/team19/lendz/AddBookFragment.java:16]: Thats sth we can clarify tomorrow

**Comment by kb-jboateng** [comment]
Changes have been pushed

File: app/src/main/java/cmput301/team19/lendz/MainActivity.java (hunk 3 above)
Comment (kylemclean) [reply app/src/main/java/cmput301/team19/lendz/MainActivity.java]: You can create a `ViewUserProfileFragment` that displays the logged in user's profile by passing their user ID to `ViewUserProfileFragment.newInstance(String userId)`.

You can get the logged in user's ID by calling `FirebaseAuth.getInstance().getCurrentUser().getUid()`.

Comment (kb-jboateng) [reply app/src/main/java/cmput301/team19/lendz/MainActivity.java]: have you taken a look at the ViewUserProfileFragment?

Comment (kylemclean) [reply app/src/main/java/cmput301/team19/lendz/MainActivity.java]: Yes, I wrote it. What is the issue?

**Pull request merged**